### PR TITLE
Ep29 create menu

### PR DESCRIPTION
**Pull request by stratfob**

### CODE REVIEW

**Comment by SeanDurban** [comment]
image
This is minor but could the add menu link be placed above the Menu text so doesnt get mixed up

File: menus/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```
Comment (SeanDurban) [inline menus/.gitignore]: Minor: Can this be moved to the other gitignore, just to keep them all in same place? Or is it there cause we have to have the folder present?

File: views/uploadMenu.ejs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<head>
+	<title>Upload Menu</title>
```
Comment (SeanDurban) [inline views/uploadMenu.ejs]: Can we change to flash messages here for the errors?

File: test/test_menuMapper.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+var assert = require('assert');
+var mapper = require('../mappers/menuMapper');
```
Comment (SeanDurban) [inline test/test_menuMapper.js]: Depending  on whether everyone agrees with my changes to the DB testing (Running these tests just wiped my DB again >:o ) Then we could reflect this here. Should be only minor changes!

File: menus/.gitignore (hunk 1 above)
Comment (stratfob) [reply menus/.gitignore]: Yeah the folder has to be present, it didn't work having it in the other gitignore because git wasn't keeping the empty folder.

Comment (SeanDurban) [reply menus/.gitignore]: Yep that makes sense then! Did you try 'menu/*' ?  Think that will just ignore contents

File: views/uploadMenu.ejs (hunk 2 above)
Comment (SeanDurban) [reply views/uploadMenu.ejs]: I tried to upload a non pdf file and ended up on a page just saying the error message and had to make my way back through url

**Pull request merged**